serverAkkaHttp/src/main/scala/RPCRouter.scala

```diff
@@ -35,7 +36,11 @@ trait Router extends RPCServer with PathMacro with MetaDataMacro with LazyLoggin
 
   def exceptionHandler = ExceptionHandler {
     case e: FailException[_] =>
-      logger.error(s"encoding error: ${e.getMessage}")
+      //logging left
+      e.response.entity match {
+        case HttpEntity.Strict(_, data) => logger.error(s"${e.response.status.value} ${data.utf8String}")
+        case complexEntity => logger.error(s"${e.response.status} ${e.response.entity}")
```

Comment:
in case request is too complex to be parsed log whatever scala decide to `toString`

Reply:
`e.response.status` -> `e.response.status.value` for consistency with the previous case